MMM-Screencast.js

```diff
@@ -16,6 +16,10 @@ Module.register("MMM-Screencast", {
 		this.sendSocketNotification('SET_CONFIG', this.config);
 	},
 
+	stop: function() {
```

Comment:
So whats the point of this? Besides messaging the user that they're stopping the MM and this module?